products/appengine/api.yaml

```diff
@@ -67,30 +67,74 @@ objects:
           - ALLOW
           - DENY
   - !ruby/object:Api::Resource
-    name: 'Service'
+    name: 'ServiceSplitTraffic'
     description: |
       A Service resource is a logical component of an application that can share state and communicate in a secure fashion with other services. 
       For example, an application that handles customer requests might include separate services to handle tasks such as backend data analysis or API requests from mobile devices. 
       Each service has a collection of versions that define a specific set of code used to implement the functionality of that service.
     base_url: 'apps/{{project}}/services'
-    self_link: 'apps/{{project}}/services/{{id}}'
+    create_url: 'apps/{{project}}/services/{{service_id}}?migrateTraffic={{migrate_traffic}}&updateMask=split'
+    create_verb: :PATCH
+    update_url: 'apps/{{project}}/services/{{service_id}}?migrateTraffic={{migrate_traffic}}'
+    update_verb: :PATCH
+    update_mask: true
     references: !ruby/object:Api::Resource::ReferenceLinks
-      guides:
-        'Official Documentation':
-          'https://cloud.google.com/appengine/docs/admin-api/deploying-overview'
       api: 'https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services'
+    async: !ruby/object:Api::Async
+      operation: !ruby/object:Api::Async::Operation
+        kind: 'appengine#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/global/operations/{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::Async::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::Async::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::Async::Error
+        path: 'error/errors'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::Boolean
+        name: 'migrate_traffic'
+        url_param_only: true
+        description: |
+          If set to true traffic will be migrated to this version.
     properties:
       - !ruby/object:Api::Type::String
         name: 'name'
-        output: true
         description: |
           Full path to the Service resource in the API. Example apps/myapp/services/default.
           This field is used in responses only. Any value specified here in a request is ignored.
       - !ruby/object:Api::Type::String
         name: 'id'
```

Comment:
We have some liberty in terms of what we can call this field because it doesn't actually correspond to any fields we care about in the API. What do you think of `service` instead of `id`/`service_id`?

Reply:
wdyt here?